static/js/app.js

```diff
@@ -800,15 +763,16 @@ $(function () {
             }
         },
         getCurrentUrlParams: function () {
-          var dateRange = app.model.get("dateRange");
-          var center = app.map.getCenter();
-          return "start_date=" + moment(dateRange[0]).format("YYYY-MM-DD") +
-              "&end_date=" + moment(dateRange[1]).format("YYYY-MM-DD") +
-              "&show_fatal=" + (parseInt(app.model.get("showFatal")) || 0) +
-              "&show_severe=" + (parseInt(app.model.get("showSevere")) || 0) +
-              "&show_light=" + (parseInt(app.model.get("showLight")) || 0) +
-              "&show_inaccurate=" + (parseInt(app.model.get("showInaccurateMarkers")) || 0) +
-              "&zoom=" + app.map.zoom + "&lat=" + center.lat() + "&lon=" + center.lng();
+        // TODO: After the filters a re ready improve that too accordingly
```

Comment:
Fixed - shortened the line to only contain date range, zoom, and geo location.

If we would like to allow the user to save a filter configuration and send it in a link, we should do it in a different issue, and generate a unique code for each configuration that would be parsed to bounding box query.